hourly_watch.py

```diff
@@ -15,6 +15,50 @@
 
 
 # Ska-specific watchers
+class SkaURLWatch(JobWatch):
+    def __init__(self, task, maxage_hours, url=None,):
+        self.type = 'URL'
+        self.basename = url
+        super(SkaURLWatch, self).__init__(task, url, maxage=maxage_hours * HOURS)
+
+    @property
+    def headers(self):
+        if not hasattr(self, '_headers'):
+            try:
+                import urllib2
```

Comment:
Sorry, I should have started with the comment to use `requests` instead of `urllib2`.  You can consider `requests` to be required component of Ska.  From there, you are guaranteed to have a `headers` attribute in the `Response` object.  I'm not sure from the [docs](http://docs.python-requests.org/en/latest/api/#requests.Response) if `headers` can be None or must always be a "case-insensitive dict object".